### PR TITLE
Add codesandbox links to samples

### PR DESCRIPTION
**Pull request by Noah-Silvera**
- Added links to open each sample in codesandbox
- Edited the samples to work in codesandbox where appropriate, in particular, the iframe sample.

### CODE REVIEW

**Comment by Noah-Silvera** [comment]
@ianschmitz the codesandbox links currently point to master. You will need to change the "master" portion of the codesandbox URL to "code.sandbox.5.7" in order to see the working sandboxes!

**Comment by Noah-Silvera** [comment]
> The first time i open a sandbox i get this 502 error, but clicking the refresh button on the internal browser fixes it once the dev server is booted up. I see a number of CORS errors in the console that may be related. Have you experienced this?:
> 
> image

I have experienced this, and you might be right in saying they are related. It looks like the CORS issues you saw are related to socketio - when I experienced It, I got an error as described in this issue https://github.com/webpack/webpack-dev-server/issues/1628
webpack dev server may not be able to communicate with the browser and refresh it. Not sure how big of a deal this is though, as I imagine anyone who is doing web dev should try refreshing the browser as one of their first debugging steps.

File: viewer/src/App.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ async function getSampleData(sampleName: string): Promise<Sample> {
 
     try {
         page = await import(
```
[the remark this answers is not in the document]
Comment (Noah-Silvera) [reply viewer/src/App.tsx]: Renamed!

[the remark this answers is not in the document]
Comment (ianschmitz) [reply viewer/src/App.tsx]: Can you also rename in the sample interface?

**Comment by ianschmitz** [comment]
> webpack dev server may not be able to communicate with the browser and refresh it. Not sure how big of a deal this is though, as I imagine anyone who is doing web dev should try refreshing the browser as one of their first debugging steps.

Indeed. It's just unfortunate that it seems to happen all the time, as it may give the impression that it's broken. I wonder if enabling the "hard refresh" option may help here?

**Comment by Noah-Silvera** [comment]
> Indeed. It's just unfortunate that it seems to happen all the time, as it may give the impression that it's broken. I wonder if enabling the "hard refresh" option may help here?

Sadly, the hard refresh option refreshes the server, but not the browser. It will do an yarn install and yarn start, but not refresh the browser window.

File: viewer/src/App.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Noah-Silvera) [reply viewer/src/App.tsx]: Done!

**Comment by Noah-Silvera** [comment]
> > Sadly, the hard refresh option refreshes the server, but not the browser. It will do an yarn install and yarn start, but not refresh the browser window.
> 
> Ah crap okay. Hmm i guess we just live with it for now then?
> 
> I'm not sure that the CORS issue is actually an issue with WDS if it resolves itself after a refresh.

Yeah, I think we have to live with it for now.
The CORS issue might be unrelated, but I think there is still issues with WDS. WDS is supposed to refresh the browser on a code change, but that doesn't happen in codesandbox.

**Pull request merged**